### DISCUSSION TITLE
Purpose of "eye" icon in bottom left of screen when viewing an uploaded photo?

### DISCUSSION THREAD

**Discussion by lxs** [tripupapp/tripup-ios#1] [Q&A] [answered]
An "eye" symbol is displayed in the bottom left of the app when viewing a photo.
What is the purpose of this setting?

**observations from initial exploration:** 
- When selecting this button it changes from white to blue; also an "eye" watermark is then displayed on this image when viewed in an album.
- There is no on-hover pop-up message, nor content in FAQ, to help explain the usage.
- Once this option is enabled the load-time of the album is considerably faster (upon opening the app: 90+ photos in album were taking 20+ seconds to fully load, yet after enabling the "eye" of just 1 photo in the album then load time of album reduced to <5 seconds).

See attached screenshot of sample photo with "eye" icon enabled, and screenshot of photo album; "eye" icon is circled in red.
![IMG_8193](https://user-images.githubusercontent.com/70903/121199430-f9136180-c883-11eb-9bee-480f4b82c70e.JPG)
![IMG_8194](https://user-images.githubusercontent.com/70903/121199465-003a6f80-c884-11eb-9f1e-58ea1e0904bf.JPG)

**Comment by vin047** [comment, marked as answer]
The eye button makes a photo visible to all users in the album. You add users to an album by tapping the person with the plus icon (top left, right next to the back button in your second screenshot).

> There is no on-hover pop-up message, nor content in FAQ, to help explain the usage.

Good point. I will add a pop-up message explaining this once a user has toggled this for the first time.

> Once this option is enabled the load-time of the album is considerably faster (upon opening the app: 90+ photos in album were taking 20+ seconds to fully load, yet after enabling the "eye" of just 1 photo in the album then load time of album reduced to <5 seconds).

This... shouldn't happen. Sounds like a rendering bug to me. If you could open up a GitHub issue (bug report) and attach a screen recording showing this, I can take a look to see what's going on.
